doc/data/messages/f/function-redefined/good.py

```diff
@@ -0,0 +1,6 @@
+def get_email():
+    pass
+
+
+def get_address():
+    pass
```

Comment:
```suggestion
```
Imo it's often due to a copy paste so the solution is often to remove completely. It's also clearer to not add another function as we don't have any context in the example.